feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -0,0 +1,378 @@
+# TE-18.1 gRIBI MPLS in UDP Encapsulation with OC QoS scheduler
+
+Create AFT entries using gRIBI to match on next hop group in a
+network-instance and encapsulate the matching packets in MPLS in UDP.
+
+Create a policy routing configuration using gNMI to decapsulate MPLS
+in UDP packets which are sent to a loopback address and apply to
+the DUT.
+
+The MPLS in UDP encapsulation is expected to follow
+[rfc7510](https://datatracker.ietf.org/doc/html/rfc7510#section-3),
+but relaxing the requirement for a well-known destination UDP port.  gRIBI is
+expected to be able to set the destination UDP port.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
```

Comment:
q: tbd w/ vendors: how do the BGP routing work w/o the PBR solution ? what if the inner dst_A belongs to CR and used to traverse BGP packets ? Currently the PBR rules intercept only echo reply packets and all other packets are encapsulated. w/ vrf there will not be any PBR rule. Not sure if this is discussed and sorted in past

Reply:
Added TODO item to specify a solution for ensuring BGP packets are matched, encapsulated and forwarding to a specified  destination using OC policy-forwarding terms.